org.radare.iaito.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: org.radare.iaito
+
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+
+command: iaito
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --allow=devel
+
+cleanup:
+  - /include
+  - /lib/debug
```

Comment:
```suggestion
```

Reply:
in /lib/debug there are all the source code of the app, should we leave it there?

Reply:
the debug symbols get separated into a different flatpak.